.circleci/config.yml

```diff
@@ -22,29 +22,29 @@ jobs:
       - run: ./bin/flake8_tests.sh
   backend-unit-tests:
     environment:
-      COMPOSE_FILE: .circleci/docker-compose.circle.yml
       COMPOSE_PROJECT_NAME: redash
-    docker:
-      - image: circleci/buildpack-deps:xenial
+    machine:
+      image: ubuntu-1604:202004-01
```

Comment:
Can you elaborate why this is changed?

Reply:
That's because Docker in docker can't mount docker volume. 
https://support.circleci.com/hc/en-us/articles/360007324514-How-can-I-use-Docker-volume-mounting-on-CircleCI-